### PR TITLE
ci: add pr comment with the deployment cluster IPs

### PR DESCRIPTION
**Pull request by Shabirmean**
### Description:
- The reviewers didn't have a way to check the deployment from the PR unless they directly go to the CI project and find IP of the correct Service.
- Thus, we needed an improvement to add a comment on the PR with the external IP of the newly created LB Service

### Changees:
- I added two extra cloudbuild steps to get the external IP and comment on the PR using the GH CLI tool.
- However, I am using a _Personal Access Token_ generated on my user account because there is no way in Github to generate a token against a repo or organization.

### CODE REVIEW

**Comment by Shabirmean** [comment]
test

**Comment by Shabirmean** [comment]
test

**Comment by Shabirmean** [comment]
test: $(cat /tmp/t.$pr.txt)

**Comment by Shabirmean** [comment]
test: t

**Comment by Shabirmean** [comment]
Access the Point-of-Sale at: "$(cat /workspace/pr-101.api-server-lb.ip)"

**Comment by Shabirmean** [comment]
Access the Point-of-Sale at: [35.232.88.13](http://35.232.88.13)

File: .github/cloudbuild/pos-pr-open-to-main.yaml [hunk 1]
```diff
@@ -93,10 +93,43 @@ steps:
...
+  - -c
+  - |
+    gh pr comment $_PR_NUMBER --repo $_HEAD_REPO_URL --body "Access the Point-of-Sale at: [$(cat /workspace/pr-$_PR_NUMBER.ip)](http://$(cat /workspace/pr-$_PR_NUMBER.ip))"
+  waitFor: [ 'get-external-loadbalancer-ip' ]
+
+availableSecrets:
+  inline:
+  - kmsKeyName: 'projects/$PROJECT_ID/locations/global/keyRings/$_GHTOKEN_KEYRING_NAME/cryptoKeys/$_GHTOKEN_KEY_NAME'
+    envMap:
+      GITHUB_TOKEN: 'CiQAuVI6Z93rj1B1R4yer3G1VzEmW7aHtjATP2jOtyJjkW5qOR8SUQBHWet7zXRDJQTBSvFZuXXsOBIxtvf0VDM610Tjgb5XJY8hbIuBxhsMfDy3+/p2ojVnYmE+Mg9NDg5FGZhlJlB/+PBMCumkbMovTXD582u17A=='
```
[the remark this answers is not in the document]
Comment (Shabirmean) [reply .github/cloudbuild/pos-pr-open-to-main.yaml:132]: - Will this be available for the CloudBuild in GCP? I think this only works with Github actions.
- Also this is using an encrypted token via [Cloud Key Management Service](https://cloud.google.com/build/docs/securing-builds/use-encrypted-credentials) in GCP
- So Cloudbuild decrypts it before supplying it to the builder image. 
- No one can decrypt it without access to the [decryption key in KMS](https://pantheon.corp.google.com/security/kms/keyring/manage/global/pos-github-token-ring/key?project=point-of-sale-ci)

[the remark this answers is not in the document]
Comment (bourgeoisor) [reply .github/cloudbuild/pos-pr-open-to-main.yaml:132]: This looked like a raw GitHub token; my bad! Carry on!

**Review by ahrarmonsur — APPROVED** [review]
LGTM!

**Pull request merged**